src/odemis/driver/test/xt_client_test.py

```diff
@@ -712,6 +711,72 @@ def test_apply_autofocus(self):
         autofocus_state = self.microscope.is_autofocusing(channel)
         self.assertEqual(autofocus_state, False)
 
+    def test_get_mpp_orientation(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        mpp_orientation = self.microscope.get_mpp_orientation()
+        self.assertIsInstance(mpp_orientation, float)
+        # TODO K.K. check range with Wilco
+        self.assertTrue(0 < mpp_orientation < 360)
+
+    def test_get_aperture_index(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        aperture_index = self.microscope.get_aperture_index()
+        self.assertIsInstance(aperture_index, float)
+        self.assertTrue(0 < aperture_index < 14)
+
+    def test_set_aperture_index(self):
+        """
+        Simple test, checks if no error occur during calling the method.
+        """
+        # TODO Check if setting of the value actually happens and if an int value is also allowed.
+        self.microscope.set_aperture_index(10)
+        aperture_index = self.microscope.get_aperture_index()
+        self.assertEqual(aperture_index, 10)
```

Comment:
I suggest to set another one (0 should work as this is the one for multibeam). Then you can check the aperture actually changed compared to the previous one. In case 10 was already set before you run the test, you would not notice if the set call did not work properly.

Also I suggest you do it the same way as the other tests: 

- start with getting the currently set value
- then change to a different value and check it is set
- then put back the originally set value